docs/user-guide.md

```diff
@@ -41,9 +41,14 @@ This guide provides detailed instructions on how to import and configure your Li
 
 ### Settings
 
-Liberty Tools for IntelliJ IDEA honours  the following external settings:
-- Maven home path set in the **Preferences > Build, Execution, Deployment > Build Tools > Maven** window when running Liberty dev mode on Maven projects.
-- Gradle JVM set in the **Preferences > Build, Execution, Deployment > Build Tools > Gradle** window when running Liberty dev mode on Gradle projects.
+Liberty Tools for IntelliJ IDEA honors the following settings:
+
+#### IntelliJ settings:
+- Maven home path set in the **Settings > Build, Execution, Deployment > Build Tools > Maven** window when running Liberty dev mode on Maven projects.
+- Gradle JVM set in the **Settings > Build, Execution, Deployment > Build Tools > Gradle** window when running Liberty dev mode on Gradle projects.
+
+#### Environment Variables:
+- **JAVA_HOME** in the current terminal for running Liberty.
```

Comment:
what does "running liberty" mean in this context? Does it mean the **JAVA_HOME** value that is set in a `server.env` file for a given instance of Liberty? The answer might be obvious for the user but this seems a bit vague

Reply:
Ah, to be more clear, the **JAVA_HOME** system environment variable (not server.env) is used to specify which Java to use when running Liberty dev mode.